tests/unit/moduleapi/subcommands.tcl

```diff
@@ -21,9 +21,26 @@ start_server {tags {"modules"}} {
 
     test "Module pure-container command fails on arity error" {
         catch {r subcommands.bitarray} e
-        assert_match {*wrong number of arguments*} $e
+        assert_match {*wrong number of arguments* *subcommands.bitarray*} $e
 
         # Subcommands can be called
         assert_equal [r subcommands.bitarray get k1] {OK}
     }
+
+    test "Module get current command fullname" {
+        assert_equal [r subcommands.parent_get_fullname] {subcommands.parent_get_fullname}
+    }
+
+    test "Module get current subcommand fullname" {
+        assert_equal [r subcommands.sub get_fullname] {subcommands.sub|get_fullname}
+    }
+
+    test "COMMAND LIST FILTERBY MODULE" {
+        set reply [r command list filterby module subcommands]
+        assert_equal [lsort $reply] {subcommands.bitarray subcommands.bitarray|get subcommands.bitarray|set subcommands.parent_get_fullname subcommands.sub subcommands.sub|get_fullname}
+    }
+
+    test "Unload the module" {
+        assert_equal {OK} [r module unload subcommands]
+    }
```

Comment:
very good you did that.
did valgrind pass?

Reply:
yes, it passed. i also wanted to do that in every module test. WDYT?

Reply:
i guess it can't harm, if you have the energy..

Reply:
i handle it in https://github.com/redis/redis/pull/10127/commits/74810de580798658d209ec7c204d61fb04fba869 and https://github.com/redis/redis/pull/10127/commits/ada89a1d6719c8ea2aa683e2d007d34d5dab6019